open_academy/models/session.py

```diff
@@ -7,6 +7,6 @@ class Session(models.Model):
     start_date = fields.Date()
     duration = fields.Integer()
     number_of_seats = fields.Integer()
-    instructor_id = fields.Many2one('res.partner')
+    instructor_id = fields.Many2one('res.partner', domain='["|", ("is_instructor","=",True), ("teacher_type","!=",False)]')
     course_id = fields.Many2one('course', required=True)
     attendees_ids = fields.Many2many('res.partner', string="Attendees")
```

Comment:
Remove the string, and test if the label on the form still appears as Attendees AFAIK the default string will be attendees and the string is not necessary please test it.

Reply:
done